packages/theme-patternfly-org/package.json

```diff
@@ -70,7 +70,7 @@
     "webpack-merge": "^5.0.9"
   },
   "peerDependencies": {
-    "react": ">=16.8.0",
-    "react-dom": ">=16.8.0"
```

Comment:
Let's update these to match @patternfly-react/react-core so we're maintaining support for 16.8+ while adding support for 17+
- https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/package.json#L60-L61